ad/movement/ad-cs/certificate-templates.md

```diff
@@ -61,6 +61,14 @@ The current default value is `0x18` (`0x8` and `0x10`). Schannel doesn't support
 
 If some certificate authentication issues are encountered in an Active Directory, [Microsoft has officially suggested](https://support.microsoft.com/en-us/topic/kb5014754-certificate-based-authentication-changes-on-windows-domain-controllers-ad2c23b0-15d8-4340-a468-4d4f3b188f16) to set the `CertificateMappingMethods` value to `0x1f` (old value).
 
+#### Issuance policies
+
+It is possible to apply issuance policies to certificate templates. This takes the form of a certificate extension, and is stored as an OID (object identifier) in the `msPKI-Certificate-Policy` attribute of the template. When the CA issues the certificate, the policy is added to the "Certificate Policies" attribute of the certificate. A template stores required policies in the `msPKI-RA-Policies` attribute.
+
+An issuance policy can be set up by a company, for example, for access control: a system can require a user to present a certificate with a given policy in order to guarantee that the system only grants access to authorised users. Finally, all issuing policies are AD object of the class `msPKI-Enterprise-Oid` stored in the PKI OID container.
```

Comment:
Finally, all issuing policies are AD object of the class `msPKI-Enterprise-Oid` stored in the PKI OID container.
 --> 
Issuing policies are `msPKI-Enterprise-Oid` objects found in the PKI OID container (`CN=OID,CN=Public Key Services,CN=Services`, in the Configuration Naming Context).